tests/ert_tests/ensemble_evaluator/prefect/test_prefect_ensemble.py

```diff
@@ -116,26 +116,25 @@ def test_run_prefect_ensemble_exception(evaluator_config, poly_ensemble):
     assert evaluator._ensemble.get_status() == state.ENSEMBLE_STATE_FAILED
 
 
-def function_that_fails_once(coeffs):
-    # all retries load pickled function --> difficult to avoid assuming filesystem
-    # Assumes sum(coeffs) is unique per realization
-    sum_coeffs = sum(coeffs.values())
-    run_path = Path("ran_once" + str(sum_coeffs))  # Avoid data races
-    if not run_path.exists():
-        run_path.touch()
-        raise RuntimeError("This is an expected ERROR")
-    try:
-        run_path.unlink()
-    except FileNotFoundError:
-        # some other real beat us to it
-        pass
-    return {"function_output": []}
-
-
 @pytest.mark.timeout(60)
 def test_prefect_retries(
     evaluator_config, function_ensemble_builder_factory, tmpdir, ensemble_size
 ):
+    def function_that_fails_once(coeffs):
+        # all retries load pickled function --> difficult to avoid assuming filesystem
+        # Assumes sum(coeffs) is unique per realization
```

Comment:
```suggestion
        # Assumes sum(coeffs) is unique per realization
        # Needs to be inside local test-scope to avoid flakyness
```
Kan vi legge til ein kommentar for ettertida, så minnast vi på dette